pep-XXXX.rst

```diff
@@ -0,0 +1,258 @@
+provided in the libraries distributed with CentOS 5, a Linux distribution
+released in April 2007. In practice, this means that Python wheels which conform
+to this policy should function on almost any linux distribution released after
+this date.
+
+
+Compilation and Tooling
+=======================
+
+To support the compilation of wheels meeting the ``manylinux1`` standard, we
+provide initial drafts of two tools.
+
+The first is a Docker image based on CentOS 5.11, which is recommended as an
+easy to use self-contained build box for compiling  ``manylinux1`` wheels.
+Compiling on a more recently-released linux distribution will generally
+introduce dependencies on too-new versioned symbols. The image comes with a
+full compiler suite installed (``gcc``, ``g++``, and ``gfortran`` 4.8.2) as
+well as the latest releases of Python and pip.
+
+The second tool is a command line executable called ``auditwheel``. First, it
+inspects all of the ELF files inside a wheel to check for dependencies on
+versioned symbols or external shared libraries, and verifies conformance with
+the ``manylinux1`` policy. This includes the ability to add the new platform
+tag to conforming wheels.
+
+In addition, ``auditwheel`` has the ability to automatically modify wheels that
+depend on external shared libraries by copying those shared libraries from
+the system into the wheel itself, and modifying the appropriate RPATH entries
+such that these libraries will be picked up at runtime. This accomplishes a
+similar result as if the libraries had been statically linked without requiring
+changes to the build system.
+
+Neither of these tools are necessary to build wheels which conform with the
+``manylinux1`` policy. Similar results can usually be achieved by statically
+linking external dependencies and/or using certain inline assembly constructs
+to instruct the linker to prefer older symbol versions, however these tricks
+can be quite esoteric.
+
+
+Platform Detection in ``pip``
+=============================
+
+TODO How does ``pip`` detect that it's running on a ``manylinux1`` compatible
+system?
+
+
+Security Implications
+=====================
+
+One of the advantages of dependencies on centralized libraries in Linux is
+that bugfixes and security updates can be deployed system-wide, and
+applications which depend on on these libraries will automatically feel the
+effects of these patches when the underlying libraries are updated. This can
+be particularly important for security updates in packages communication
+across the network or cryptography.
+
+``manylinux1`` wheels distributed through PyPI that bundle security-critical
+libraries like OpenSSL will thus assume responsibility for prompt updates in
+response disclosed vulnerabilities and patches. This closely parallels the
+security implications of the distribution of binary wheels on Windows that,
+because the platform lacks a system package manager, generally bundle their
+dependencies.
```

Comment:
We might want to note here explicitly that openssl in particular does not have a stable ABI and so cannot be included in the `manylinux1` profile .